best-practices/README.md

```diff
@@ -189,10 +189,9 @@ Email
 
 JavaScript
 ----------
-* Use Coffeescript, ES6 with [babel], or another language that compiles to
-  JavaScript
+* Use JavaScript with [babel], or another language that compiles to JavaScript.
```

Comment:
I explicitly left out mentioning ES6 as babel already supports some ES7 features. If I know numbers, they will continue to go up.